src/lib/falcor/FalcorRouter.js

```diff
@@ -93,14 +93,27 @@ export default class FalcorRouter extends BaseRouter.createClass([
       });
     },
   },
-  /* TODO: finish this
   {
-    route: "authorsBySlug[{keys:slugs}]['teams']",
+    route: "authorsBySlug[{keys:slugs}]['teams'][{integers:indices}]",
     get: (pathSet) => {
-      
+      return new Promise((resolve) => {
+        db.authorTeamQuery(pathSet.slugs).then((data) => {
+          const results = [];
+          _.each(data, (teamSlugArray, authorSlug) => {
```

Comment:
I believe, and just googled, that _.each is old syntax that may still be a working alias, but we have used forEach throughout the code already, and it's also consistent with ES6 which I'm sure is why lodash changed the syntax, so just change _.each to _.forEach